sdks/python/apache_beam/ml/inference/api.py

```diff
@@ -55,5 +54,9 @@ class RunInference(beam.PTransform):
   def __init__(self, model_loader: base.ModelLoader):
     self._model_loader = model_loader
 
+  @classmethod
+  def create(cls, model_loader_provider, **kwargs):
```

Comment:
Is this an standard xlang interface that users would expect? I don't really understand it or how it would work.

What do you think about adding a comment to explain why this exists.

Reply:
Added a comment.